### DISCUSSION TITLE
Superset 508 Fixes

### DISCUSSION THREAD

**Discussion by ErrolBlake** [apache/superset#25215] [General]
[Superset 508 Fixes.pdf](https://github.com/apache/superset/files/12542121/Superset.508.Fixes.pdf)

Good Day, 

We have attached 508 fixes to this discussion topic. We have two questions:

1. Can these 3 Errors/Issues be fixed in the next release of Superset? 
2. If the answer to question 1 is no. Then, can we fix the errors/issues and create a pull request for Superset to include in the next release?

**Comment by nytai** [comment]
I would suggest going with 2. Open a Pull Request and if it merges, keep an eye out on the next release candidate to see if it also includes your fix
